src/shared/components/Table/Table.module.css

```
@@ -130,11 +129,19 @@
 /* ==================== Cell Styling ====================*/
 .cell {
 	vertical-align: middle;
-	background-color: color-mix(in srgb, var(--foreground) 4%, transparent);
+	background-color: var(--table-cell-bg, color-mix(in srgb, var(--foreground) 4%, transparent));
+	font-size: var(--table-cell-font-size, inherit);
 	word-break: break-word;
```

Stylelint (17.13.0)

[error] Deprecated keyword "break-word" for property "word-break" (declaration-property-value-keyword-no-deprecated)

(declaration-property-value-keyword-no-deprecated)